src/Umbraco.Examine/IUmbracoIndexConfig.cs

```diff
@@ -0,0 +1,12 @@
+using Examine;
+
+namespace Umbraco.Examine
+{
+    public abstract class IUmbracoIndexConfig
```

Comment:
A class shouldn't not be prefixed with "I", that is reserved for interfaces. Seems like this should be an interface instead?